pyopencl/cl/pyopencl-complex.h

```diff
@@ -274,11 +274,19 @@
     } \
   } \
 
+#if PYOPENCL_COMPLEX_ENABLE_EXTENDED_ALIGNMENT
+#define _PYOPENCL_COMPLEX_ALIGNMENT(TYPE) 2*sizeof(TYPE)
```

Comment:
I feel like C reserves identifiers with a leading underscore for the implementation. Could you look up those rules?

Reply:
Done in 23b064efa8e557d8f2a102640506afb531e10fa2. Added a link to C docs in the commit.